src/DatePicker/index.js

```diff
@@ -68,10 +72,11 @@ export default class DatePicker {
         <button type="button" class="date-picker__toggle" aria-label="Toggle Date Picker">Toggle Date Picker</button>
 
         <div class="date-picker__calendar" style="display: none">
-          <p class="calendar__header" role="heading"></p>
           <button id="calendar__prev" type="button">${this.props.previousLabel}</button>
           <button id="calendar__next" type="button">${this.props.nextLabel}</button>
```

Comment:
When I was trying this with VO, these buttons read "Next Button" and "Previous Button". Would it be an improvement if these read, "Go to Next Month, Button" or something along those lines?